### ISSUE TITLE
native lands

### ISSUE THREAD

**Issue by mattbuys** [tdwg/dwc-qa#190]
I'm planning to assign native land designations for my occurrences (using www.native.land.ca as source).  As far as I can tell, no suitable DWC field for this kind of metadata?

**Comment by tucotuco** [comment]
@mattbuys No, there isn't a field specifically fo that, but you could include the information in [dwc:dynamicProperties](https://dwc.tdwg.org/terms/#dwc:dynamicProperties) with a key:value pair. For example, { "isNativeLand": true }.
